src/Xrd/XrdConfig.hh

```diff
@@ -94,7 +95,7 @@ static const char  *TraceID;
 XrdNetSecurity     *Police;
 XrdTcpMonInfo      *tmoInfo;
 const char         *myProg;
-const char         *myName;
+std::string         myName;
```

Comment:
It does not seem to make sense to me to replace one `const char*` with `std::string`, while leaving all the rest as `const char*`.

Reply:
I would be more than happy to clean it all up, I just wasn't sure if a surgical approach was preferred.